client/containers/DashboardSubmissions/VerdictDialog.tsx

```diff
@@ -0,0 +1,154 @@
+import React, { useState } from 'react';
+import { Callout, Button, Dialog, Classes, Checkbox } from '@blueprintjs/core';
+
+import { apiFetch } from 'client/utils/apiFetch';
+import { DefinitelyHas, Pub, DocJson } from 'types';
+import { Icon } from 'components';
+import { getEmptyDoc } from 'client/components/Editor/utils/doc';
+import WorkflowTextEditor from '../DashboardSubmissionWorkflow/WorkflowTextEditor';
+
+require('./verdictDialog.scss');
+
+type Props = {
+	isOpen: boolean;
+	onClose: () => unknown;
+	actionTitle: string;
+	completedName: string;
+	apiMethod: string;
+	status?: string;
+	initialEmailText?: DocJson;
+	pub: DefinitelyHas<Pub, 'submission'>;
+	onJudgePub: (pubId: string, status?: string) => void;
+};
+
+type PreSubmissionBodyProps = {
+	handleSubmission: (customEmailText: DocJson, shouldSendEmail: boolean) => void;
+	isHandlingSubmission: boolean;
+	actionTitle: string;
+	onClose: () => unknown;
+	initialEmailText?: DocJson;
+};
+
+const PreSubmissionBody = (props: PreSubmissionBodyProps) => {
+	const [shouldSendEmail, setShouldSendEmail] = useState(!!props.initialEmailText);
+	const [customEmailText, setCustomEmailText] = useState(props.initialEmailText || getEmptyDoc());
+	return (
+		<>
+			<div className={Classes.DIALOG_BODY}>
+				<p>Would you like to {props.actionTitle} this submission?</p>
```

Comment:
`props.actionTitle.toLowerCase()` so the action doesn't appear capitalized here.